Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:22.04
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && \
+    apt-get install -y texlive-latex-base texlive-fonts-recommended texlive-fonts-extra && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+WORKDIR /workspace
+CMD ["pdflatex", "--version"]
```

Comment:
Instead of calling pdflatex directly, you may want to call a script (that you make) instead.
This would help with a couple things:
 - You can fix the permissions/ownership of the output to match the input.
 - You can also double compile to get references resolved.